### PR TITLE
Upgrade to Emotion v10

### PR DESCRIPTION
**Pull request by etairi**
`react-emotion` package is no longer maintained, and it is recommended to use `@emotion/core` now (i.e., Emotion v10).

### CODE REVIEW

**Comment by etairi** [comment]
Done.

**Comment by etairi** [comment]
Because prior to Emotion 10 one was using the `className` prop, but now the prop name used is `css`. So, I tried to make it consistent with Emotion. You can read more about the migration [here](https://emotion.sh/docs/migrating-to-emotion-10).

**Comment by GuillaumeCisco** [comment]
Does keeping `className` prop still work?

**Comment by GuillaumeCisco** [comment]
In my personal opinion, I think it is better to evolve to the css prop with warning user of the breaking change.
Modifying a className props to a css prop is not a big refacto even in large codebases

**Comment by davidhu2000** [comment]
Ah, that make sense. Given we are requiring emotion 10 for this latest version. sounds good.

File: README.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ Each loader accepts a `loading` prop as a boolean. The loader will not render an
  
 ```js
 import React from 'react';
-import { css } from 'react-emotion';
+import { css } from '@emotion/core';
```
Comment (davidhu2000) [inline README.md]: if we are no longer importing from `react-emotion`, do we still need that package?

Comment (davidhu2000) [reply README.md]: according to the upgrade guide, i don't think we need it anymore? 

https://emotion.sh/docs/migrating-to-emotion-10

Comment (etairi) [reply README.md]: No, you don't. Now the package name is `@emotion/core`.

Comment (davidhu2000) [reply README.md]: cool, can you remove it from package.json? 

Also, please make sure to open `docs/index.html` in chrome, and run `npm run watch` and make sure all the loaders render properly

**Comment by davidhu2000** [comment]
@etairi cool, can you remove `react-emotion` from package.json?

Also, please make sure to open docs/index.html in chrome, and run npm run watch and make sure all the loaders render properly

**Comment by etairi** [comment]
Removed `react-emotion`.

**Comment by etairi** [comment]
@davidhu2000 Thanks for the info. I just noticed that, and upgraded the packages and did some minor modifications. Now it seems to work fine. I opened a new pull request #50 .

**Pull request merged**